controllers/apply/awards-for-all/fields.js

```diff
@@ -1873,8 +2026,17 @@ module.exports = function fieldsFor({ locale, data = {} }) {
             isRequired: false,
             schema: Joi.string()
                 .allow('')
+                .max(FREE_TEXT_MAXLENGTH.large)
                 .optional(),
-            messages: []
+            messages: [
+                {
+                    type: 'string.max',
+                    message: localise({
+                        en: `The organisation trading name must be ${FREE_TEXT_MAXLENGTH.large} characters or less`,
```

Comment:
`Organisation's day-to-day name must be ${FREE_TEXT_MAXLENGTH.large} characters or less`